app.py

```diff
@@ -9,13 +12,39 @@
 def index():
     return render_template('index.html', items=sorted(session.get_items(), key=lambda item: item['status'], reverse=True))
```

Comment:
You were expected to change these existing pages, and ultimately delete session_items.py, rather than continue to support two separate systems.